examples/Gero.purs

```diff
@@ -0,0 +1,33 @@
+-- Examples of how to use the Gero wallet interface through `Contract`
+--
+-- To run: `npm run dev` and visit `localhost:4008` in your browser. Make sure
+-- that you have the Gero browser extension installed. Allow the page to access
+-- your wallet when prompted by Gero
+--
+-- NOTE: Gero has the same limitations as Nami, i.e., Chromium-based only
+
+module Examples.Gero (main) where
+
+import Contract.Prelude
+import Contract.Address (getWalletAddress, getWalletCollateral)
+import Contract.Monad
+  ( Contract
+  , defaultContractConfig
+  , runContract_
+  )
+
+import Effect.Aff (launchAff_)
+
+main :: Effect Unit
+main = launchAff_ $ do
+  cfg <- defaultContractConfig
+  runContract_ cfg $ do
+    logAction getWalletAddress
+    logAction getWalletCollateral
+
+logAction
```

Comment:
I think this function should be inlined since it's trivial and is only used here.